ansible/roles/jupyterhub/files/jupyterhub_config_base.py

```diff
@@ -136,9 +136,13 @@
 mnt_root = os.environ.get('MNT_ROOT')
```

Comment:
Minor detail, we should add spacing between config settings and comments to make things more readable.

Reply:
@jgwerner do you mean that we should change all the lines in that config file that have a comment above?
there are many lines with same issue

Reply:
Good point, we can do that as a chore in a separate PR.